### DISCUSSION TITLE
How to prevent doctrine:migrations:diff modify a column?

### DISCUSSION THREAD

**Discussion by berkut1** [symfony/symfony#48400] [Q&A]
Hello,

I'm doing a new projects and want to use `IDENTITY` columns instead of `SERIAL` https://wiki.postgresql.org/wiki/Don%27t_Do_This#Don.27t_use_serial
However, doctrine will only support it in dbal 4.0 https://github.com/doctrine/dbal/pull/5396

So, is there a way to stop doctrine from touching some id columns when execute the command `doctrine:migrations:diff` ?

Thanks.

**Comment by stof** [comment]
This should be asked to the Doctrine project.
